src/helpers.js

```diff
@@ -1,3 +1,5 @@
 export const hardUpdate = newState => () => state => Object.assign({}, state, newState);
 export const softUpdate = fn =>
```

Comment:
The behavior in each of `hardUpdate` and `softUpdate` should be preserved, but the actual exports `hardUpdate` and `softUpdate` should result in...
```
console.log("Both `hardUpdate` and `softUpdate` are deprecated.  Please use `update` instead.");
```
Followed by the original behavior.